source/fundamentals/encrypt-fields.txt

```diff
@@ -3,3 +3,14 @@
 .. sharedinclude:: dbx/encrypt-fields.rst
 
    .. replacement:: driver-specific-content
+
+To use CSFLE with the driver, you need to have the MongoDB.Driver.Encryption 
```

Comment:
Could you hyperlink Mongo.Driver.Encryption to its nuget page? Should be this https://www.nuget.org/packages/MongoDB.Driver.Encryption